packages/tools/fetch-tool/src/fluidFetch.ts

```diff
@@ -31,6 +31,23 @@ async function fluidFetchOneFile(urlStr: string, name?: string) {
     await fluidFetchSnapshot(documentService, saveDir);
 }
 
+async function tryFluidFetchOneSharePointFile(server: string, driveItem: IOdspDriveItem) {
+    const { path, name, drive, item } = driveItem;
+    if (name.endsWith(".b") || name.endsWith(".fluid")) {
```

Comment:
It sounds like we have driveID & itemID, but file extension is not .fluid, then we fail silently, which is not ideal.
Should this check be in getSharePointSpecificDriveItem?